web/themes/joinup/prototype/sass/components/_icon.sass

```diff
@@ -35,7 +35,7 @@
 //
 // Styleguide 2.8
 
-$waat-icons-font-path: '../images/fonts/waat-icons'
+$waat-icons-font-path: '../images/fonts/waat-icons/waat-icons'
```

Comment:
Please move `fonts` in root of theme. Fonts are same asset as css, js, etc. but they are surely not images.